Source/Core/Common/Assembler/GekkoLexer.cpp

```diff
@@ -0,0 +1,791 @@
+#include "Common/Assembler/GekkoLexer.h"
```

Comment:
Missing license header.

```suggestion
// Copyright 2023 Dolphin Emulator Project
// SPDX-License-Identifier: GPL-2.0-or-later

#include "Common/Assembler/GekkoLexer.h"
```